modules/gitops-release-notes-1-8-4.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-4_{context}"]
+= Release notes for {gitops-title} 1.8.4
+
+{gitops-title} 1.8.4 is now available on {product-title} 4.10, 4.11, 4.12, and 4.13.
+
+[id="new-features-1-8-4_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, the bundled Argo CD has been updated to version 2.6.
+
+[id="fixed-issues-1-8-4_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, Argo CD was becoming unresponsive when there was an increase in namespaces and applications. This update fixes the issue by removing a deadlock. Deadlock occurs when two functions are competing for resources. Now, you should not experience crashes or unresponsiveness when there is an increase in namespaces or applications. link:https://issues.redhat.com/browse/GITOPS-2782[GITOPS-2782]
```

Comment:
I actually just cloned the ticket for this because we had already attached 2782 to the 1.9.1 advisory (and we can't reuse tickets) so the new ticket associated with the v1.8.4 release is GITOPS-3192 (https://issues.redhat.com/browse/GITOPS-3192). Sorry for the confusion, the rest of the numbers should be fine!

Reply:
@reginapizza I fixed the link. If everything looks good now, please give an approval. Thank you!